artifactory/commands/gradle/resources/jfrog.init.gradle

```diff
@@ -0,0 +1,31 @@
+def artifactoryUrl = '{{ .ArtifactoryURL }}'
```

Comment:
lets verify that the syntax here is not already defined on some go library like is found on maven